_posts/2016-07-11-july_update.markdown

```diff
@@ -13,22 +13,22 @@ Like I wrote to [the Hydra Tech list](https://groups.google.com/forum/#!searchin
 * supports the management of top-priority content types: image-based works and generic multi-file works (think IR-like use cases)
 * offers a user interface for dynamic configuration of the application (without having to touch Ruby or YAML files)
 * provides an administrative dashboard with reporting widgets
-* allows for batch import of content and metadata
+* allows for batch import of content and metadata.
 
 Here's where our feature and application work stands:
 
 * The Hydra-in-a-Box repository application is based on Sufia 7, and we worked with the community to release several beta releases of Sufia 7 and the gems upon which it depends. A 7.0.0 release is anticipated in the next week.
 * The application is multi-tenant with each tenant having its own database tables, Solr collection, Fedora namespace, and Redis namespace.
-* The application is deployable to Amazon Web Services using a number of Amazon's built-in services. See our [AWS GitHub repository](https://github.com/hybox/aws) to see our AWS CloudFormation templates.
-* The app currently supports generic multi-file works, and are coordinating with Hydra community members who are working on supporting more content types in Sufia 7, and we hope to include more robust support of content types later this summer. To build support for image-based works in the interim, we have wired up the generic work model we currently support to use the IIIF Image and Presentation APIs.
+* The application is deployable to Amazon Web Services using a number of Amazon's built-in services. Visit our [AWS GitHub repository](https://github.com/hybox/aws) to see our AWS CloudFormation templates.
+* The app currently supports generic multi-file works. We are coordinating with Hydra community members who are working on supporting more content types in Sufia 7, and we hope to include more robust support of content types later this year. To build support for image-based works in the interim, we have wired up the generic work model we currently support to use the IIIF Image and Presentation APIs.
```

Comment:
Specifically:

* what is a "generic multi-file work"?
* Spell out IIIF and link to the IIIF site